### ISSUE TITLE
Still not passing Travis...

### ISSUE THREAD

**Issue by brennanpincardiff** [CaRdiffR/nounprojectR#21]
Sadly, here is the link:
https://travis-ci.org/CaRdiffR/nounprojectR

**Comment by dokato** [comment]
As it throws some magick related error, I've checked how their travis config looks like. It's more complicated: https://github.com/ropensci/magick/blob/master/.travis.yml but maybe we could try it out?

**Comment by brennanpincardiff** [comment]
Mmm, that's very interesting @dokato. I don't understand the code there.... yet. 
Do you recommend a cut and paste and trial?
We also need to add a secret key and password for the Noun Project API - I've not worked out how to do that yet either :-(

**Comment by dokato** [comment]
Yes, my first try would be to create a dummy PR with this config copied. Could you give it a shot? We'll see what message we get.

Regarding the API secret etc, we can set some env variables in travis settings:
https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings
